scripts/run-smoke-test.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# Requires docker and the following ENV vars:
+#
+# PORT_CLIENT_ID
+# PORT_CLIENT_SECRET
+# PORT_BASE_URL (optional, defaults to 'https://api.getport.io')
+#
+
+SCRIPT_BASE="$(cd -P "$(dirname "$0")" && pwd)"
+ROOT_DIR="$(cd -P "${SCRIPT_BASE}/../" && pwd)"
+
+RANDOM_ID="${RANDOM}${RANDOM}" # DOGE - RANDOM
+INTEGRATION_IDENTIFIER="smoke-test-integration-${RANDOM_ID}"
+BLUEPRINT_DEPARTMENT="fake-department-${RANDOM_ID}"
+BLUEPRINT_PERSON="fake-person-${RANDOM_ID}"
+BLUEPRINT_IDENTIFIERS="${BLUEPRINT_DEPARTMENT} ${BLUEPRINT_PERSON}"
+FAKE_INTEGRATION_VERSION=${FAKE_INTEGRATION_VERSION:-0.1.1-dev}
+PORT_BASE_URL_FOR_DOCKER=${PORT_BASE_URL}
+
+if [[ ${PORT_BASE_URL} =~ localhost ]]; then
+    PORT_BASE_URL_FOR_DOCKER=${PORT_BASE_URL//localhost/host.docker.internal}
+fi
+
+TEMP_DIR=$(mktemp -d -t smoke-test-integration.XXXXXXX)
+RESOURCE_DIR_SUFFIX="integrations/fake-integration/.port/resources"
+cp -r "${ROOT_DIR}"/${RESOURCE_DIR_SUFFIX} "${TEMP_DIR}"
+
+sed -i.bak "s/fake-department/${BLUEPRINT_DEPARTMENT}/g" "${TEMP_DIR}"/resources/blueprints.json
+sed -i.bak "s/fake-person/${BLUEPRINT_PERSON}/g" "${TEMP_DIR}"/resources/blueprints.json
+sed -i.bak "s/\"fake-department\"/\"${BLUEPRINT_DEPARTMENT}\"/g" "${TEMP_DIR}"/resources/port-app-config.yml
+sed -i.bak "s/\"fake-person\"/\"${BLUEPRINT_PERSON}\"/g" "${TEMP_DIR}"/resources/port-app-config.yml
```

Comment:
should we rename the file? as its very very specific for fake-integration?

Reply:
it's hard code for the fake integration at this delivery, can work on make it generic now or as a separate PR